app/src/main/java/fr/neamar/kiss/result/ShortcutResult.java

```diff
@@ -0,0 +1,78 @@
+package fr.neamar.kiss.result;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.content.pm.PackageManager.NameNotFoundException;
+import android.content.res.Resources;
+import android.graphics.drawable.Drawable;
+import android.os.Handler;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+import fr.neamar.kiss.R;
+import fr.neamar.kiss.pojo.ShortcutPojo;
+
+public class ShortcutResult extends Result {
+    private final ShortcutPojo shortcutPojo;
+
+    public ShortcutResult(ShortcutPojo shortcutPojo) {
+        super();
+        this.pojo = this.shortcutPojo = shortcutPojo;
+    }
+
+    @Override
+    public View display(final Context context, int position, View v) {
+        if (v == null)
+            v = inflateFromId(context, R.layout.item_app);
+
+        TextView appName = (TextView) v.findViewById(R.id.item_app_name);
+        appName.setText(enrichText(shortcutPojo.displayName));
+
+        final ImageView appIcon = (ImageView) v.findViewById(R.id.item_app_icon);
+        if (position < 15) {
```

Comment:
this is useful on the loooong app list, it should not be needed for shortcuts, you can always assume the code 

```
appIcon.setImageDrawable(this.getDrawable(context));
```

will work fast.